src/Warnings.js

```diff
@@ -35,5 +35,13 @@ var WARNINGS = {
                'to the other party to let you know if they are confused about ' +
                'something, rather than implying that you "didn\'t make sense." ' +
                '--Tara Sophia Mohr', },
+    { keyword: 'I should',
+      source:  'http://www.lifehack.org/articles/communication/7-things-not-to-say-and-7-things-to-start-saying.html',
+      message: 'The word "should" is inherently negative. "Should" implies a lose: lose ' +
+              'situation and it\'s just not conducive to positive outcomes in life. ' + 
+              'It\'s a form of criticism, and it\'s best left out of your everyday language. ' + 
+              'Instead of beating yourself up for what you should have done, ' + 
+              'focus on what you have the power to change.' + 
+              '-- Zoe B', },
```

Comment:
These messages (all of them, not just this one) need to include recommended replacements for the offending word(s).